assets/Languages/en-US.cfg

```diff
@@ -647,6 +647,7 @@ SECURITY_N = N/A
 SECURITY_O = N/A
 SECURITY_P = N/A
 CAMERA_INTERIOR = Switches to the train's interior view
+CAMERA_NO_INTERIOR = Switches to the train's interior view. However, the panel is not displayed.
```

Comment:
**CAMERA_INTERIOR_NOPANEL** would probably be a better name, certainly makes more sense in English :)

Reply:
Thanks!
I fixed it. Please check it.